src/cache/SortKeyCache.ts

```diff
@@ -22,7 +22,7 @@ export interface SortKeyCache<V> {
   /**
    * returns value for the key and exact blockHeight
    */
-  get(contractTxId: string, sortKey: string, returnDeepCopy?: boolean): Promise<SortKeyCacheResult<V> | null>;
+  get(key: string, sortKey: string): Promise<SortKeyCacheResult<V> | null>; //FIXME: why get by two values and put by object??
```

Comment:
please fix -> use the object

Reply:
Done, but I'm having second thoughts on this change...

Reply:
mind sharing? :>